### ISSUE TITLE
Module/Class Length being checked even when not enabled

### ISSUE THREAD

**Issue by thewatts** [bbatsov/rubocop#2273]
I may be missing something -
The feature isn't enabled, but it's still being run against my code.

Config:
```yaml
# .rubocop.yml
Metrics/ClassLength:
  Description: Avoid classes longer than 100 lines of code.
  Enabled: false
  CountComments: false
  Max: 100
```

Error:
```
components/users/spec/managers/users/session_manager_spec.rb:3:1: C: Module has too many lines. [101/100]
module Users
```

Thanks for a great gem!

**Comment by jonas054** [comment]
Run with the `-D` option. You'll see which cop is reporting.

**Comment by thewatts** [comment]
@jonas054 OH GOSH! I forgot to reply to this!

Thanks so much for your help!!!!
